src/content/1.7/modules/concepts/hooks/list-of-hooks.md

```diff
@@ -1788,10 +1788,10 @@ displayOrderConfirmation
     Parameters:
     ```php
     array(
-      'total_to_pay' => (float) Total amount with tax,
-      'currency' => (string) Currency sign,
-      'objOrder' => (object) Order,
-      'currencyObj' => (object) Currency
+      'order' => (object) Order,
```

Comment:
This is actually the only parameter sent since 1.7.0 (https://github.com/PrestaShop/PrestaShop/commit/d8c85492832b980e1395184fd3f835a88bf48166)